Admin/Factory/FieldFactory.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace LAG\AdminBundle\Admin\Factory;
+
+use LAG\AdminBundle\Admin\Configuration\ApplicationConfiguration;
+use LAG\AdminBundle\Admin\Field;
+use LAG\AdminBundle\Admin\FieldInterface;
+use BlueBear\BaseBundle\Behavior\ContainerTrait;
+use Exception;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Field factory. Instances fields with its renderer.
+ */
+class FieldFactory
+{
+    use ContainerTrait;
```

Comment:
You may name your Trait ContainerAwareTrait, also, class that used this trait should implements the ContainerAwareInterface.